integration/mediation-tests/tests-mediator-2/src/test/resources/artifacts/ESB/mediatorconfig/transform/transformMediatorSchema.xml

```diff
@@ -0,0 +1,95 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<definitions xmlns="http://ws.apache.org/ns/synapse">
+    <proxy name="transformMediatorSimple" startOnLoad="true" transports="https http">
```

Comment:
Shall we add this definition as individual files which get deploys in the startup itself so that test running time reduces.

Reply:
done